lib/src/wrappers/ModalWrapper.jsx

```diff
@@ -66,6 +67,17 @@ export default class ModalWrapper extends PureComponent {
     open: false,
   }
 
+  static getDerivedStateFromProps(nextProps) {
+    // only if accept = true close the dialog
+    if (nextProps.isAccepted) {
```

Comment:
So you're handling opening/closing of Modal declaratively, using `isAccepted` as the signal to close, correct?  As opposed to imperatively using a ref to ModalWrapper?

Reply:
@mcMickJuice you are exactly right

Reply:
cool! thanks